sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -712,7 +712,7 @@ case class AlterTableDropPartition(
 /**
  * The logical plan of the ALTER TABLE ... RENAME TO PARTITION command.
  */
-case class AlterTableRenamePartition(
+case class RenamePartition(
```

Comment:
Even now, we allow to rename only one partition but Hive supports renaming of multiple partitions in one command. Maybe Spark will support this too in the future. Should we name it as `RenamePartitions`?

Reply:
yea, to be consistent with ADD and DROP